docs-source/spring/content/development/cli/_index.md

```diff
@@ -559,4 +558,182 @@ The following is an example development workflow using the CLI:
        
          oractl:>config list --service-name myserv --service-profile obaas --service-label 0.1 --property-key ktest2
          400 : "Couldn't find any property for submitted query."
-         ```
+        ```
+
+8. Use the `GraalVM Compile Commands` to:
+
+* Upload a **.jar** file to the Oracle Backend for Spring Boot and Microservices and its GraalVM compiler service.
+* Start a compilation of your microservice to produce an executable native file **.exec**.
+* Retrieve the last logs available regarding a compilation in progress or terminated.
+* Download the **.exec** file to deploy on the backend.
+* Purge the files remaining after a compilation on the remote GraalVM compiler service.
+
+The GraalVM Compile Commands are the following:
+
+```cmd
+oractl:>help 
+   
+GraalVM Compile Commands
+       compile-download: Download the compiled executable file.
+       compile: Compile a service with GraalVM.
+       compile-purge: Delete a launched job.
+       compile-logs: Compilation progress.
+```
+
+   1. Use the `compile` command to upload and automatically start compilation using the following command:
+
+```cmd
+   oractl:>help compile
+       NAME
+       compile - Compile a service with GraalVM.
+
+       SYNOPSIS
+              compile [--artifact-path String] --help 
+
+       OPTIONS
+              --artifact-path String
+              Service jar to compile location
+              [Mandatory]
+
+              --help or -h 
+              help for compile
+              [Optional]
+```
+
+   Because the compilation of a **.jar** file using the tool `native-image` does not support cross-compilation, it must be on the same kind of platform where the application will run. This service guarantees a compilation in the same operating system and CPU type where the service will be executed on the Kubernetes cluster.
+
+   The Spring Boot application **pom.xml** with the plugin:
+
+```xml
+<plugin>
+       <groupId>org.graalvm.buildtools</groupId>
+       <artifactId>native-maven-plugin</artifactId>
+</plugin>
+```
+
+The project should be compiled on the developer desktop with GraalVM version 22.3 or later using an **mvn** command. For example:
+  
+```cmd
+  mvn -Pnative native:compile -Dmaven.test.skip=true
+```
+
+   This pre-compilation on your desktop checks if there are any issues on the libraries used in your Spring Boot microservice. In addition, your executable **.jar** file must include ahead-of-time (AOT) generated assets such as generated classes and JSON hint files. For additional information, see [Converting Spring Boot Executable Jar](https://docs.spring.io/spring-boot/docs/current/reference/html/native-image.html#native-image.advanced.converting-executable-jars).
```

Comment:
Question: Should Microservice always be init capped? Sometimes it is and sometimes it isn't.

Reply:
It should only have an initial captial when it is part of the product name "Oracle Backend for Spring Boot and Microservices" or the start of a sentence.

Reply:
Got it. Thanks!